1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
```

Comment:
"Configuring an HTTP Proxy ..." or "Configuring HAProxy ..."

("HAProxy" is a name, whereas "HTTP proxy" really is just a noun, so we need an article in the latter case)